R/Rules-methods.R

```diff
@@ -2030,6 +2090,12 @@ setMethod(
     mtd_cv <- (mad(mtd_samples) / median(mtd_samples)) * 100
     do_stop <- (mtd_cv <= stopping@thresh_cv) && (mtd_cv >= 0)
 
+    report_label <- if (isTRUE(stopping@report_label == "")) {
+      paste("P(MTD >", x@thresh, "* NBD | P(DLE) = ", x@target, ") \u2265 ", x@prob)
```

Comment:
same here: what is the `x` here? I do not see this object on the list of explicit arguments to this function.